packages/hoppscotch-common/src/components/collections/Properties.vue

```diff
@@ -125,12 +134,12 @@ watch(
     if (props.show) {
       persistenceService.setLocalConfig(
         "unsaved_collection_properties",
-        JSON.stringify(<EditingProperties>{
+        JSON.stringify({
```

Comment:
both the angle-bracket syntax and using `as` to assert just tells typescript that we know what we're doing, and dont bother checking it. but in the future if someone passes an invalid object that does not adhear to the type, typescript won't catch that.

ideally in this case, we can extract the object into variable and assign a type for that, and tries to fix any errors that comes up.